src/modules/csm/state.py

```diff
@@ -169,7 +164,7 @@ def migrate(
         else:
```

Comment:
We don't need this else branch anymore, I think. We can just call `_migrate_frames_data`.